docs/release-notes/2.21.0.md

```diff
@@ -0,0 +1,56 @@
+- [PMM-7477](https://jira.percona.com/browse/PMM-7477): Support custom TLS certificates when monitoring remote MongoDB instances
+- [PMM-7888](https://jira.percona.com/browse/PMM-7888): Custom TLS certificates allow SSL connections to monitored PostgreSQL instances (e.g. on Heroku) (Thanks to Jyoti Prakash for reporting this issue)
+
+## Improvements
+
+- [PMM-8267](https://jira.percona.com/browse/PMM-8267): Backup Management: Active progress indicator
+- [PMM-8549](https://jira.percona.com/browse/PMM-8549): Backup Management: Show loading status on delete window
+- [PMM-8542](https://jira.percona.com/browse/PMM-8542): Backup Management: Inform that times should be entered in UTC timezone format
+- [PMM-8316](https://jira.percona.com/browse/PMM-8316): DBaaS: PSMDB 1.9 operator support
+- [PMM-8386](https://jira.percona.com/browse/PMM-8386): Overview text on Home page omits mentioning PostgreSQL is supported
+- [PMM-7612](https://jira.percona.com/browse/PMM-7612): Integrated Alerting: Validate 'From' email address format (Settings → Communication → Email) 
+- [PMM-7570](https://jira.percona.com/browse/PMM-7570): Specify Custom Basic Auth password for Agents when adding Services
+- [PMM-8560](https://jira.percona.com/browse/PMM-8560): Add support for Debian 11 ("Bullseye") to `pmm-client` package
+- [PMM-7087](https://jira.percona.com/browse/PMM-7087): Rename and highlight that PostgreSQL custom queries example file is overwritten on upgrade (Thanks to Daniel Guzman Burgos for reporting this issue)
+- [PMM-8568](https://jira.percona.com/browse/PMM-8568): Use latest CentOS patches for creating OVA, AMI and Azure images
+- [PMM-5291](https://jira.percona.com/browse/PMM-5291): Update ClickHouse version from 19.7.5.27 to 21.3-lts
+- [PMM-8091](https://jira.percona.com/browse/PMM-8091): Collect and present additional ProxySQL metrics taken from `runtime_mysql_servers` table
+
+## Bugs Fixed
+
+- [PMM-8458](https://jira.percona.com/browse/PMM-8458): Backup Management: Inconsistent auto-fill of 'Vendor' field with on-demand backup
+- [PMM-8404](https://jira.percona.com/browse/PMM-8404): Dashboard image rendering plugin renders image that includes error message
+- [PMM-7286](https://jira.percona.com/browse/PMM-7286): Query Analytics can't handle colon character (`:`) in service names (Thanks to Daniel Guzman Burgos for reporting this issue)
+- [PMM-7278](https://jira.percona.com/browse/PMM-7278): `mongo_exporter` fails to authorize when MongoDB running with `authMechanism=MONGODB-X509` (Thanks to Lena D for reporting this issue)
+- [PMM-8307](https://jira.percona.com/browse/PMM-8307): Default config limits for allowed connections prevents monitoring large numbers (500+) of DB servers
+- [PMM-2168](https://jira.percona.com/browse/PMM-2168): `rds_exporter` not stopped when all RDS instances are removed or disabled
+- [PMM-8219](https://jira.percona.com/browse/PMM-8219): PMM Server update panel "Current version" empty if no internet connectivity
```

Comment:
```suggestion
- [PMM-8219](https://jira.percona.com/browse/PMM-8219): PMM Server update panel "Current version" fails to display if no internet connectivity
```

Reply:
The display is `Current version: ()`. There isn't strictly a failure or error message. It just doesn't print anything. I don't want to suggest a cause but merely report the fact of the ticket.